resources/asciidoctor/spec/added_spec.rb

```diff
@@ -40,4 +38,50 @@
     DOCBOOK
     expect(actual).to eq(expected.strip)
   end
+
+  it "inline version creates a phrase" do
+    actual = convert <<~ASCIIDOC
+      == Example
+      words added:[some_version]
+    ASCIIDOC
+    expected = <<~DOCBOOK
+      <chapter id="_example">
+      <title>Example</title>
+      <simpara>words <phrase revisionflag="added" revision="some_version"/>
+      </simpara>
+      </chapter>
+    DOCBOOK
+    expect(actual).to eq(expected.strip)
+  end
+
+  it "inline version creates a phrase with extra text if provided" do
+    actual = convert <<~ASCIIDOC
+      == Example
+      words added:[some_version, more words]
+    ASCIIDOC
+    expected = <<~DOCBOOK
+      <chapter id="_example">
+      <title>Example</title>
+      <simpara>words <phrase revisionflag="added" revision="some_version">
+        more words
+      </phrase>
+      </simpara>
+      </chapter>
+    DOCBOOK
+    expect(actual).to eq(expected.strip)
+  end
+
+  it "inline version is not invoked without the ::" do
```

Comment:
Given the PR description... does this match the behavior of `asciidoc` too?

Reply:
So, in asciidoc the inline macro *is* invoked without the `:`, but in asciidoctor it isn't. We "fix" the input to add the `:` in the compat preprocessor. This is a "doesn't it work how I think it should" test more than anything.